src/C++/FileStore.cpp

```diff
@@ -60,7 +67,7 @@ FileStore::FileStore( std::string path, const SessionID& s )
 
   try
   {
-    open( false );
+    open( deleteFile );
   }
```

Comment:
Parameter from constructor now used.  Default parameter value is 'false' making change non invasive